nlptest/nlptest.py

```diff
@@ -657,3 +675,25 @@ def pass_custom_bias_data(file_path: str, test_name: str = None, append: bool =
             data = json.load(f)
 
         TestFactory.call_add_custom_bias(data, test_name, append)
+
+    def edit(self, output_path: str, **kwargs):
+        """ testcases is exported to a csv file and can be edited. 
+        The edited file can be imported back to the harness"""
+        temp_df = self.testcases()
+        temp_df = temp_df[temp_df['category'].isin(['robustness', 'bias'])]
+        temp_df.to_csv(output_path, index=False)
+
+        return self
+
+    def import_testcases(self, input_path: str, **kwargs):
```

Comment:
we can rename to " import_edited_testcases "

Reply:
I'm done